demos/c_api_minimal_app/Makefile

```diff
@@ -20,22 +20,24 @@ HTTPS_PROXY := "$(https_proxy)"
 NO_PROXY := "$(no_proxy)"
 BASE_OS_TAG_UBUNTU ?= 20.04
 BASE_OS_TAG_REDHAT ?= 8.7
-REDHAT_PACKAGE_URL ?="https://github.com/openvinotoolkit/model_server/releases/download/v2022.3/ovms_redhat.tar.gz"
-UBUNTU_PACKAGE_URL ?="https://github.com/openvinotoolkit/model_server/releases/download/v2022.3/ovms_ubuntu.tar.gz"
 
 OVMS_CPP_DOCKER_IMAGE ?= openvino/model_server
 OVMS_CPP_IMAGE_TAG ?= latest
 BASE_OS ?= ubuntu
 DIST_OS ?= $(BASE_OS)
 
 ifeq ($(BASE_OS),ubuntu)
-  PACKAGE_URL = ${UBUNTU_PACKAGE_URL}
+  ifeq ($(BASE_OS_TAG_UBUNTU),20.04)
+	DLDT_PACKAGE_URL ?= http://s3.toolbox.iotg.sclab.intel.com/ov-packages/l_openvino_toolkit_ubuntu20_2023.1.0.11298.1588a332173_x86_64.tgz
+  else ifeq  ($(BASE_OS_TAG_UBUNTU),22.04)
+	DLDT_PACKAGE_URL ?= http://s3.toolbox.iotg.sclab.intel.com/ov-packages/l_openvino_toolkit_ubuntu22_2023.1.0.11298.1588a332173_x86_64.tgz
```

Comment:
Did you test it? You replace ovms package with ov - how does it work?
Second thing is that it will stop working for users outside of org.